examples/README.md

```diff
@@ -0,0 +1,137 @@
+## Snap Plugin Go Library Examples
+Here you will find example plugins that cover the basics for writing collector, processor, and publisher plugins in the examples folder.
+
+## Build Plugins & Use with Snap
+
+To get these example collector, processor, and publisher plugins to build properly and work with Snap you will need to have [glide](https://glide.sh/) installed on your $GOPATH.
+
+Also to test these plugins with Snap, you will need to have [Snap](https://github.com/intelsdi-x/snap) installed, check out these docs for [Snap setup details] (https://github.com/intelsdi-x/snap/blob/master/docs/BUILD_AND_TEST.md#getting-started).
+
+###1. Get the plugin library repo:
+`go get github.com/intelsdi-x/snap-plugin-lib-go` will add the repo to your $GOPATH
+
+note: if you want to contribute to this repository you should fork the snap-plugin-lib-go repo and open a PR.
```

Comment:
Is the intention of the note to let people know they can contribute or to warn them to fork before they open a PR?